src/models/movies-model.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const db = require('../db');
+
+class MoviesModel {
+
+  async getList() {
+      const query = {
+        text: 'SELECT * FROM movie ORDER BY year ASC;',
+        values: [],
+      }
+      const res = await db.query(query);
+
+      return res.rows;
+  }
+
+  async getMovieById(id) {
+    const query = {
+      text: 'SELECT * FROM movie WHERE id = $1;',
+      values: [id],
+    };
+    const res = await db.query(query);
+
+    return res.rows[0];
+  }
+
+  async getMoviesTitles(year) {
+    const yearString = year ? 'WHERE year = $1' : '';
+    const query = {
+      text: `SELECT title FROM movie ${yearString} ORDER BY year ASC;`,
+      values: year ? [year]: [],
+    };
+    const res = await db.query(query);
+
+    return res.rows.map(item => item.title).join('\n');
+  }
+
+  async addMovie(newMovie) {
+    const { title, year, imdbRating } = newMovie;
+    const query = {
+      text: 'INSERT INTO movie(title, year, imdbrating) VALUES($1, $2, $3) RETURNING *;',
+      values: [title, year, imdbRating],
+    }
```

Comment:
should be semicolon after constant declaration

Reply:
> should be semicolon after constant declaration

fixed